pages/journey-list/index.js

```diff
@@ -10,6 +10,10 @@ export default function JourneyList({
   result,
   id,
   onToggleFavourite,
+  selectedEntry,
+  setSelectedEntry,
```

Comment:
Where do `selectedEntry` and `setSelectedEntry` come from?
No such props are passed down in `_app.js`. Remove them here.